com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md

```diff
@@ -49,10 +49,10 @@ New Materials in HDRP use the [Lit shader](Lit-Shader.md) by default. To create
 | **Smoothness UV Scale Transform** | Sets the tiling rate (xy) and offsets (zw) for the Smoothness Mask Map. |
 | **Smoothness Min**                | Set the minimum smoothness for this Material.                |
 | **Smoothness Max**                | Set the maximum smoothness for this Material.                |
-| **Specular Color**                | Set the color of the primary specular highlight.             |
-| **Specular Multiplier**           | Modifies the **Specular Color** by this multiplier.          |
+| **Specular Color**                | Set the representative color of the highlight which will be used to drive both the primary specular highlight color which is mainly monochrome and the secondary specular highlight color which is chromatic.|
```

Comment:
Set the representative color of the highlight that Unity uses to drive both the primary specular highlight color, which is mainly monochrome, and the secondary specular highlight color, which is chromatic.